webapp/src/main/webapp/vue-apps/login-internal-onboarding/components/InternalOnboardingResetForm.vue

```diff
@@ -173,6 +177,33 @@ export default {
     toggleConfirmShow() {
       this.showConfirmPassword = !this.showConfirmPassword;
     },
-  },
+    submitForm() {
+      let url = `${eXo.env.portal.context}/${eXo.env.portal.selectedNodeUri}?`;
+      for (const [key, value] of new URLSearchParams(window.location.search)) {
+        url += `${key}=${value}&`;
+      }
+      const body = `action=resetPassword&username=${encodeURIComponent(this.username)}&password=${encodeURIComponent(this.password)}&password2=${encodeURIComponent(this.confirmPassword)}&captcha=${encodeURIComponent(this.captcha)}`;
```

Comment:
Use `FormData.append`+ `.toString`, see it [here](https://github.com/Meeds-io/social/blob/develop/webapp/src/main/webapp/vue-apps/common/js/SpaceService.js#L193-L194)

Reply:
done for this file. Also applied to other concerned vue component